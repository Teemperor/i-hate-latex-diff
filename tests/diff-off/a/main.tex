
\documentclass[12pt]{article}
\begin{document}
Prefix
Text
Old
Unchanged
Text
\end{document}
