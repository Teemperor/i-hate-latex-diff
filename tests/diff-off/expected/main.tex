
\documentclass[12pt]{article}
\begin{document}
Prefix
%diff-off
Text
New
%diff-on
Unchanged
Text
\addedChange{Marked}
\end{document}
